unison-src/transcripts/fix2254.output.md

```diff
@@ -64,24 +64,29 @@ unique type A a b c d
 Let's do the update now, and verify that the definitions all look good and there's nothing `todo`:
 
 ```ucm
-.a2> update.old
+.a2> update
 
-  ⍟ I've updated these names to your new definition:
-  
-    type A a b c d
+  Okay, I'm searching the branch for code that needs to be
+  updated...
+
+  That's done. Now I'm making sure everything typechecks...
+
+  Everything typechecks, so I'm saving the results...
+
+  Done.
 
 .a2> view A NeedsA f f2 f3 g
 
   type A a b c d
-    = B b
+    = A a
```

Comment:
Some re-ordering of constructor names here is strange, not sure if it's reason for concern.

Reply:
Hm, I'm not sure either